addons/l10n_vn/models/account_move.py

```diff
@@ -0,0 +1,13 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_vn_edi_invoice_number = fields.Char(
```

Comment:
```suggestion
    l10n_vn_e_invoice_number = fields.Char(
```
I know it's the name of the field in the EDI pr, but this will be cleaner.
I'll adapt the EDI one later